Updated Japanese translation

KabanFriends opened the pull request:
I fixed few mistranslations/typos in Japanese translation file.
I also checked how it looks in the actual app (built by myself).



But... I found some strange things in the font...
For example, look at this screenshot:
![image](https://user-images.githubusercontent.com/45000995/54768103-ddd7bf80-4c41-11e9-8b41-b13055a292cb.png)

Look at characters called Hiragana and Katakana (あいう and アイウ)
and Kanjis (設定)
They look like they are using different fonts. (Hiragana and Katakana(あいうアイウ) looks nice but Kanjis(設定) looks too "clean" like Times New Roman font)

So could you change the font, at least when it shows Japanese? ...Or add setting to select the font?

If you're going to change the font itself, I recommend using Meiryo font (It's included in Japanese Windows 10 (I don't know about english windows) and I think its not hard to find/get that font)

There's another problem because of Japanese incompatible font
![image](https://user-images.githubusercontent.com/45000995/54768869-7de21880-4c43-11e9-85b5-8870f55d523c.png)
Kanjis in this picture: 接続 使用 解放 必要
the first red marked texts (接続) are completely same character but lower one has weird italic effect, but it has no special formats

This should be fixed by using a new font
or at least please add a setting to select the font (maybe the list of installed fonts)

KabanFriends commented on app/locale/ja.ts:
```diff
@@ -32,58 +32,58 @@ export default {
   noticeUserRequestJoin:
     '<Username id="{{userId}}">{{username}}</Username> が参加申請を送りました。',
   offline: 'オフライン',
   ok: 'OK',
```
Note: "OK" is used by Japanese people too

KabanFriends commented:
![image](https://user-images.githubusercontent.com/45000995/54770025-ca2e5800-4c45-11e9-9577-14e961230ae2.png)
some more pictures
this difference exists in official release version too
first 4 characters (ひらがな) are Hiraganas
last 2 are kanjis
you can clearly see pixels in kanji but its hard to see them in hiragana

KabanFriends commented:
*misclosed*
im sorry

AndreasWebdev commented:
@samuelmaddock 
I've been working on japanese content before, this was my softfallback font-family for japanese content (Just add the "normal" fonts before those)..

`font-family: 'Mplus 1p', 'Hiragino Sans', 'Hiragino Kaku Gothic Pro', '游ゴシック' , '游ゴシック体' , YuGothic , 'Yu Gothic', 'ＭＳ ゴシック' , 'MS Gothic', sans-serif;`

samuelmaddock commented:
Really appreciate the effort @KabanFriends. The font used in Metastream is called [Karla](https://fonts.google.com/specimen/Karla). I'll see if I can make any improvements with fonts, otherwise I'll look into having a font preference based on language selection.

@AndreasWebdev good to know, I'll use that list if I can't fix up the Karla font.

KabanFriends commented:
well there is an updated Japanese translation file that needs to be merged
you should merge it before this request will be forgotten

KabanFriends commented:
ah misclose again

KabanFriends commented:
there's another small problem but this one depends on the environment and only happens when you build it in unsupported environment (so this problem does NOT happen in already-built official releases)

![image](https://user-images.githubusercontent.com/45000995/54821018-555e2b00-4ce4-11e9-959e-cac6ea1849ed.png)
left one: fixed language list
right one: broken language list 
The reason of that problem was flag codes, so removing flag code characters will fix the problem.

the index file itself was already broken since I cloned it into my computer (I added japanese in the list after cloning)
![image](https://user-images.githubusercontent.com/45000995/54821061-7fafe880-4ce4-11e9-9464-58b72603c970.png)

but I'm not sure those flag codes are compatible with japanese characters (like putting flag codes and japanese characters in the same file) because that problem started to happen in japanese environment.

So I recommend to remove flag codes and replace them with other things like "(US)" to prevent more problems to happen.

samuelmaddock commented:
Going to merge the translation changes and track the font rendering issue in #83 83
